hip-service/Migrations/20200110085309_intial.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace hip_service.Migrations
+{
+    public partial class intial : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4796734)